src/antennachecker/README.md

```diff
@@ -27,5 +27,16 @@ will be generated to indicate violated nets. 4 APIs are provided as an interface
 ### C++ commands
 
  - `PAR_max_wire_length(dbNet * net, int routing_level)`
-   This function returns the max wire length allowed to add for a given net, in a
-   specific layer.
+
```

Comment:
I don't think there is much point in documenting c++ "commands".
Put them in the code at the top of the function or the header file so they
are maintained if the function changes.